### PR TITLE
Add partition names to Session and TaskOptions

### PR DESCRIPTION
**Pull request by lemaitre-aneo**

### CODE REVIEW

File: Protos/V1/objects.proto [hunk 1]
```diff
@@ -15,6 +15,7 @@ message TaskOptions {
 	google.protobuf.Duration max_duration = 2;
 	int32 max_retries = 3;
 	int32 priority = 4;
+  string partition = 5;
```
Comment (jfonseca-aneo) [inline Protos/V1/objects.proto:18]: partition_id?

Comment (wkirschenmann) [reply Protos/V1/objects.proto]: I'd say `partition_id`

Comment (jfonseca-aneo) [reply Protos/V1/objects.proto]: Ok; let's leave as partition

Comment (lemaitre-aneo) [reply Protos/V1/objects.proto]: `partition_id` it is, then

**Review by jfonseca-aneo — APPROVED** [review]

**Pull request merged**